### ISSUE TITLE
ssl certificate testing keys 

### ISSUE THREAD

**Issue by ohari5336** [walkor/Workerman#529]
hello owner
I try your this library it's good but when I set up wss I got 
certificate invalid error then I add my site certificate (.ca) file
then it shows error: 

SSL handshake error: stream_socket_enable_crypto(): Unable to set private key file `/home/ohari5336 /domains/test.ohari5336.in/public_html/wss/ssl.pem'

please share a working wss certificate file for testing.

**Comment by light-source** [comment]
```
$context         = [
			'ssl' => [
				'local_cert'  => 'PATH_TO_SSL_FULLCHAIN_FILE',
				'local_pk'    => 'PATH_TO_SSL_PRIVATE_KEY_FILE',
				// required line, else can't connect in chrome
				'verify_peer' => false,
			],
		];
		$wsWorker = new Worker( 'websocket://0.0.0.0:' . 'YOUR_PORT', $context );
		$wsWorker->transport = 'ssl';
```
Try this code and don't forget replace strings to real data

**Comment by walkor** [comment]
> Unable to set private key file `/home/ohari5336 /domains/test.ohari5336.in/public_html/wss/ssl.pem'

Seems `/home/ohari5336 /domains/test.ohari5336.in/public_html/wss/ssl.pem` not exist or it is not a valid  private key.

If the problem persists, try using nginx proxy SSL like this.
```config
server {
  listen 443;

  ssl on;
  ssl_certificate /path/of/your.pem;
  ssl_certificate_key /path/of/your.key;
  ssl_session_timeout 5m;
  ssl_session_cache shared:SSL:50m;
  ssl_protocols SSLv3 SSLv2 TLSv1 TLSv1.1 TLSv1.2;
  ssl_ciphers ALL:!ADH:!EXPORT56:RC4+RSA:+HIGH:+MEDIUM:+LOW:+SSLv2:+EXP;

  location /wss
  {
    proxy_pass http://127.0.0.1: YOUR_PORT;
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "Upgrade";
    proxy_set_header X-Real-IP $remote_addr;
  }
  #....
}
```

Then the client side code like this.
```js
ws = new WebSocket("wss://your_site.com/wss");
```

**Comment by Endy-c** [comment]
I got an `SSL handshake error: stream_socket_enable_crypto(): Handshake Timeout`, and temporary resolved by nginx proxy.
However, I don't figure out what exactly happened to the program. WSS works fine on two servers using PHP 8. Worked fine on PHP7.4.4 server before, and suddenly got the error yesterday after I restart worker. I've tried to change cert and key, both of them worked well under nginx but fail under PHP7.4.4. Mark it here as a conundrum.
